cpp/src/arrow/ipc/writer.h

```diff
@@ -75,6 +76,19 @@ struct WriteStats {
   /// Number of replaced dictionaries (i.e. where a dictionary batch replaces
   /// an existing dictionary with an unrelated new dictionary).
   int64_t num_replaced_dictionaries = 0;
+
+  /// initial and serialized (may be compressed) body lengths for record batches
+  /// these values show the total sizes of all record batch body lengths
```

Comment:
Will this also include the dictionary batches?

Reply:
yes